index.html

```diff
@@ -242,10 +248,72 @@ <h3 class="text-green-dark">Multi-Post Stories Gain+Glory</h3>
             </div>
           </li>
         </ul>
+        <!-- Work pop-up mobile -->
+        <div id="wpc-mobile" class="js-container work-popup-container hide">
+          <div class="work-popup-bg">
+            <a class="work-popup-close close-button">
+              <img
+                src="images/icons/cancel-button-mobile.svg"
+                alt="cancel-button"
+              />
+            </a>
+            <div class="work-popup-content-container">
+              <img class="work-popup-img" src="images/backgrounds/snapshoot-portfolio.svg" alt="work thumbnail" />
+              <h3 class="text-blue font-600-32-44">dummy-text</h3>
+              <ul class="work-tag work-tag-popup">
+                <li>dummy-text</li>
+              </ul>
+              <p class="text-blue-2">dummy-text</p>
+              <div class="work-popup-buttons">
+                <button type="button" class="button-1 project-live">
+                  <span>See Live</span>
+                  <img src="images/icons/see-live.svg" alt="see-live-icon">
+                </button>
+                <button type="button" class="button-1 project-source">
+                  <span>See Source</span>
+                  <img src="images/icons/github-white.svg" alt="see-source-icon">
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- Work pop-up desktop -->
+        <div id="wpc-desktop" class="js-container work-popup-container hide">
+          <div class="work-popup-bg close-button">
+            <div class="work-popup-content-container">
+              <div class="work-popup-close-container">
+                <a class="work-popup-close">
+                  <img
+                    src="images/icons/cancel-button-desktop.svg"
+                    alt="cancel-button"
+                  />
+                </a>
+              </div>
+              <img class="work-popup-img" src="images/backgrounds/snapshoot-portfolio-desktop.svg" alt="work thumbnail" />
+              <div class="work-popup-title_buttons">
+                <h3 class="text-blue font-700-32-44">dummy-text</h3>
+                <div class="work-popup-buttons">
+                  <button type="button" class="button-1 project-live">
+                    <span>See Live</span>
+                    <img src="images/icons/see-live.svg" alt="see-live-icon">
+                  </button>
+                  <button type="button" class="button-1 project-source">
+                    <span>See Source</span>
+                    <img src="images/icons/github-white.svg" alt="see-source-icon">
+                  </button>
+                </div>
+              </div>
+              <ul class="work-tag work-tag-popup">
+                <li>dummy-text</li>
+              </ul>
+              <p class="text-blue-2 text-popup-desktop">dummy-text</p>
+            </div>
+          </div>
+        </div>

```

Comment:
- [ ] [REQUIRED] Hello @JauJau088 , You have done an incredible job 👍🏽 However, you need to ensure that the project section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). Kindly remove the created HTML code for the projects list.

Reply:
Thanks for the feedback. Updated!